pkg/password/password_policies.go

```diff
@@ -0,0 +1,167 @@
+package password
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"unicode/utf8"
+)
+
+var defaultSpecialCharacters = "!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~"
```

Comment:
Please prefix unexported global vars with `_`. They are much easier to spot then.